Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Token/Evm.cs

```diff
@@ -0,0 +1,178 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using ChainSafe.Gaming.Evm.Providers;
+using ChainSafe.Gaming.Evm.Transactions;
+using ChainSafe.Gaming.UnityPackage.Model;
+using ChainSafe.Gaming.Web3;
+using Nethereum.Hex.HexConvertors.Extensions;
+using Nethereum.Hex.HexTypes;
+using Nethereum.Signer;
+using Nethereum.Util;
+using UnityEngine;
+using ChainSafe.Gaming.Marketplace;
+
+namespace Scripts.EVM.Token
+{
+    public static class Evm
+    {
```

Comment:
@rob1997 you can see which of these we have in Web3Unitys class. I think you should maybe just leave the GetNonce (or maybe, we can remove it from the samples all toghether). I did remove the evm class completely since I don't want people to reference it in 3.0 at all.

Reply:
There's no reference to `EVM` class in `Web3Unity.cs`

Reply:
Requested changes made